ui/src/app/job-list/job-list.component.ts

```diff
@@ -133,6 +151,10 @@ export class JobListComponent implements OnInit {
     this.reloadJobs(this.route.snapshot.queryParams['q']);
   }
 
+  handleColumnsChanged() {
```

Comment:
What is the purpose of this function? You seem to already be binding `this.displayFields` into the child component in the template